libc/src/__support/float_to_string.h

```diff
@@ -290,7 +290,7 @@ LIBC_INLINE cpp::UInt<MID_INT_SIZE> get_table_negative(int exponent, size_t i) {
     } else {
       ten_blocks = 0;
       five_blocks = i;
-      shift_amount = static_cast<int>(shift_amount + (i * BLOCK_SIZE));
+      shift_amount = shift_amount + (static_cast<int>(i) * BLOCK_SIZE);
```

Comment:
I think you can remove the parens here now; operator precedence

here and below, there seems to be excess parens?

Reply:
personally I prefer to have parentheses when an equation is ambiguous. Even if operator precedence does the correct thing, it's less cognitive load to clarify with parentheses.